vetiver/ptype.py

```diff
@@ -80,7 +80,7 @@ def vetiver_create_ptype(data):
 @vetiver_create_ptype.register
 def _(data: pd.DataFrame):
     """
-    Create ptype for a pandas dataframe
+    Create input data prototype for a pandas dataframe
```

Comment:
Good catch! The word model is a bit overloaded. I moved this into `types.py` and used the language create_prototype and Prototype rather than create_model/BaseModel 👍